vignettes/fims-demo.Rmd

```diff
@@ -0,0 +1,432 @@
+---
+title: "Introducing the Fisheries Integrated Modeling System (FIMS)"
+output: github_document
+---
```

Comment:
There is no engine listed so I get the following error when running `devtools::build_vignette()`
```
Warning message:
In tools::buildVignettes(...) :
  Files named as vignettes but with no recognized vignette engine:
   'vignettes/fims-demo.Rmd'
(Is a VignetteBuilder field missing?)
```
suggested code stolen from fims-path-maturity.Rmd
```
vignette: >
  %\VignetteIndexEntry{FIMS Path: Maturity}
  %\VignetteEngine{knitr::rmarkdown}
  %\VignetteEncoding{UTF-8}
```